src/Plugins/ComplexCore/src/ComplexCore/Filters/CropImageGeometry.cpp

```diff
@@ -396,9 +348,12 @@ IFilter::PreflightResult CropImageGeometry::preflightImpl(const DataStructure& d
     zMax = zMin + 1;
   }
 
-  tDims[0] = (xMax - xMin) + 1;
-  tDims[1] = (yMax - yMin) + 1;
-  tDims[2] = (zMax - zMin) + 1;
+  // The ImageGeometryDimensions go from Fastest to Slowest, XYZ.
+  geomDims[0] = (xMax - xMin) + 1;
+  geomDims[1] = (yMax - yMin) + 1;
+  geomDims[2] = (zMax - zMin) + 1;
+
+  std::vector<usize> dataArrayShape = {geomDims[2], geomDims[1], geomDims[0]}; // The DataArray shape goes slowest to fastest (ZYX)
 
   FloatVec3 targetSpacing;
   targetSpacing = srcSpacing;
```

Comment:
@imikejackson Why do we create the new variable `targetSpacing` here if we just assign it the same as `srcSpacing`.  Could we just use `srcSpacing` directly instead of creating an extra variable.

Reply:
Maybe just rename the variable to `spacing` ?

Reply:
fixed